lib/i18n/backend/active_record.rb

```diff
@@ -74,6 +76,11 @@ def lookup(locale, key, scope = [], options = {})
             key = key[0..-2]
           end
 
+          if ActiveRecord.config.cache_translations
+            keys = [locale.to_sym] + key.split(I18n::Backend::Flatten::FLATTEN_SEPARATOR).map(&:to_sym)
```

Comment:
I think this way it should be cleaner since we call `to_sym` twice
```suggestion
            keys = ([locale] + key.split(I18n::Backend::Flatten::FLATTEN_SEPARATOR)).map(&:to_sym)
```